File: app/src/main/java/com/artmurka/artmurkaapp/views/fragments/interfaces/IFragmentCategoryChild.java
package com.artmurka.artmurkaapp.views.fragments.interfaces;

import com.artmurka.artmurkaapp.model.pojo.itemlist.categories.Success;

import java.util.ArrayList;

public interface IFragmentCategoryChild {
    void showCategories(ArrayList<Success> categoriesList);
    void showError(String msg);
}

File: app/src/main/java/com/artmurka/artmurkaapp/android/views/fragments/interfaces/IFragmentCategoryChild.kt
package com.artmurka.artmurkaapp.android.views.fragments.interfaces

import com.artmurka.artmurkaapp.data.model.pojo.itemlist.categories.Success
import com.artmurka.artmurkaapp.presenter.PresenterView

import java.util.ArrayList

interface IFragmentCategoryChild  : PresenterView {
    fun showCategories(categoriesList: ArrayList<Success>)
    fun showError(msg: String)
}
